### DISCUSSION TITLE
Flux OS on Proxmox

### DISCUSSION THREAD

**Discussion by MCrissDev** [RunOnFlux/flux#1399] [Q&A]
Got this when I run fluxos in proxmox: Failed: Error on FluxOs response - Unavailability to mount applications volumes. Impossible to run applications.

Any ideas?

**Comment by MorningLightMountain713** [comment]
Sounds like the user you are running FluxOS with doesn't have permissions - but could be a few things, depending on your set up. Ask in Flux Discord... people will help you there.
